packages/ui/GridListImageItem/GridListImageItem.less

```diff
@@ -25,4 +25,12 @@
 		pointer-events: none;
 		height: 100%;
 	}
+
+	.caption {
+		/* Public Class Names */
+	}
+
+	.subCaption {
```

Comment:
We can (should) combine these into one set of rules:
```
	.caption,
	.subCaption {
		/* Public Class Names */
	}
```
They just need to exist in the file, with _some_ content in them, so they will be available in the hash when imported and won't be pruned out by webpack. Currently, the CSS comment is the trick middle-spot between being pruned and available.